klog_test.go

```diff
@@ -567,6 +567,7 @@ func TestOpenAppendOnStart(t *testing.T) {
 	if !strings.Contains(string(b), x) {
 		t.Fatalf("got %s, missing expected Info log: %s", string(b), x)
 	}
+	os.Remove(f.Name())
```

Comment:
can this be:

`defer os.Remove..`

after this line:
`f, err := ioutil.TempFile("", "test_klog_OpenAppendOnStart")`

?

Reply:
My initial thought was as your suggestion, but i guess it would be better that we can get some hints from this temproary log file when test failed

Reply:
it's better to always remove it, otherwise the test is leaking files on disk.
if the test is failing the developer can comment the line to debug.

Reply:
Thanks, updated